client/src/components/InfoCard.tsx

```diff
@@ -1,77 +1,89 @@
-import { useState } from "react";
-import {
-  Avatar,
-  Badge,
-  Box,
-  Button,
-  Card,
-  CardBody,
-  CardFooter,
-  CardHeader,
-  ChakraProvider,
-  Flex,
-  Heading,
-  IconButton,
-} from "@chakra-ui/react";
-import { Autocomplete, Option } from "chakra-ui-simple-autocomplete";
-import { BiLike, BiChat, BiShare } from "react-icons/bi";
+import { Dispatch, SetStateAction, useRef } from "react";
+import { Badge, Box, Button, Image, Textarea } from "@chakra-ui/react";
+import { BiLike, BiChat } from "react-icons/bi";
 import { StarIcon } from "@chakra-ui/icons";
-const InfoCard = () => {
-  const property = {
-    imageUrl: "https://bit.ly/2Z4KKcF",
-    imageAlt: "Rear view of modern home with pool",
-    beds: 3,
-    baths: 2,
-    title: "2445 Guilford dr, Abbotsford BC ",
-    formattedPrice: "$2,900",
-    reviewCount: 20,
-    rating: 4,
+import { Post } from "../types/Post";
+import { Comment } from "../types/Comment";
+import "styles/userHome.css";
+
+interface props {
+  post: Post;
+  comment: Comment;
+  setComment: Dispatch<SetStateAction<Comment>>;
+  updateComments: (e: any) => void;
```

Comment:
The correct type here should be `FormEvent`. Tho I think the correct way of doing this is:
```js
<form onSubmit={(e) => {
  e.preventDefault();
  props.updateComments();
}}>
```
and just remove argument `e` from `updateComments`. Prop functions should not have to worry about handling DOM events.